airbyte-integrations/connectors/source-google-sheets/google_sheets_source/helpers.py

```diff
@@ -135,7 +135,6 @@ def get_available_sheets_to_column_index_to_name(
         client, spreadsheet_id: str, requested_sheets_and_columns: Dict[str, FrozenSet[str]]
     ) -> Dict[str, Dict[int, str]]:
         available_sheets = Helpers.get_sheets_in_spreadsheet(client, spreadsheet_id)
-        print(f"available_sheets: {available_sheets}")
```

Comment:
```suggestion
        import logging
        logger = logging.getLogger("airbyte")
        ...
        
        logger.info(f"Available sheets: {available_sheets}") 

```

Reply:
WDYT @mdibaiee ?

Reply:
If that means the message will get logged as an AirbyteLogMessage, then yes!

I see elsewhere in code this is the pattern used:
```
AirbyteLogger().log("WARN", "Skip non-grid sheets: " + "".join(non_grid_sheets))
```